Utils/trigger_private_build.py

```diff
@@ -117,6 +126,11 @@ def main():
     # get github token parameter
     arg_parser = argparse.ArgumentParser()
     arg_parser.add_argument('--github-token', help='Github token')
+    arg_parser.add_argument('--nightly', help='Is nightly build', action=argparse.BooleanOptionalAction)
+    arg_parser.add_argument('--sdk-ref', help='Whether to override the sdk branch')
+    arg_parser.add_argument('--slack-channel', help='The slack channel in which to send the notification', default='C04CHML16P8')
```

Comment:
use the channel name and not the id, afterwards it's hard to find which one it is